ParisBNF/abb/BNFabb20.xml

```diff
@@ -34,13 +34,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <msContents> 
                         <msItem xml:id="ms_i1">
                             <locus from="3r" to="127v"/>
+                            <title type="complete" ref="LIT3171Commen"/>
                         <msItem xml:id="ms_i1.1">
                             <locus from="3r" to="125v"/>
-                            <title type="complete" ref="LIT3171Commen"/>
+                            <title type="complete">Commentary</title>
```

Comment:
please extend the title to make it understandable: here Commentary on the Hebrews by John Chrysostom
below: Biography of John Chrysostom

Reply:
Done.